File: Python/Step07/BOJ_2566.py
max_val = -1
max_row = 0
max_col = 0

for i in range(9):
    a = list(map(int, input().split()))
    for j in range(9):
        if a[j] > max_val:
            max_val = a[j]
            max_row = i
            max_col = j

print(max_val)
print(max_row+1, max_col+1)

File: Java/Step07/BOJ_2566.java

import java.util.Scanner;

public class BOJ_2566 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int maxVal = -1;
        int maxRow = 0;
        int maxCol = 0;

        for(int i = 0; i < 9; i++){

            int[] row = new int[9];
            for (int j = 0; j < 9; j++){
                row[j] =sc.nextInt();
            }
            int rowMax = row[0];
            int rowMaxCol = 0;

            for (int j= 1; j<9; j++){
                if (row[j]>rowMax) {
                    rowMax = row[j];
                    rowMaxCol = j;
                }
            }
            if (rowMax > maxVal){
                maxVal = rowMax;
                maxRow = i;
                maxCol = rowMaxCol;
            }
        }
        System.out.println(maxVal);
        System.out.println((maxRow+1) + " " + (maxCol + 1));
        sc.close();
    }
    
}
